index.bs

```diff
@@ -1385,38 +1383,49 @@ Build a composed graph up to a given output operand into a computational graph a
   <summary>
     The <dfn method for=MLGraphBuilder>build(|outputs|)</dfn> method steps are:
   </summary>
-    1. Let |promise| be [=a new promise=].
+    1. If |outputs| is empty, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
```

Comment:
It's an interesting idea to consider what it would mean for a graph to have inputs but *no outputs*. I have seen some custom operators in models which emitted no outputs because they existed purely for other side effects, or they might exist for debugging (to print out or dump results). Within WebNN though there is no such thing anyway, and so this check makes sense 👍. The reverse is certainly true though that a graph can have outputs with no inputs. e.g. A graph with the `fillSequence` operator (renamed to a `constant` overload later) which fills a tensor with a sequence and has no input. The same would apply to random number generator operators too.  (resolve me)